c7n/filters/health.py

```diff
@@ -29,12 +29,20 @@ class HealthEventFilter(Filter):
         types={'type': 'array', 'items': {'type': 'string'}},
         statuses={'type': 'array', 'items': {
             'type': 'string',
-            'enum': ['open', 'upcoming']
+            'enum': ['open', 'upcoming','closed']
         }})
 
     permissions = ('health:DescribeEvents', 'health:DescribeAffectedEntities',
                    'health:DescribeEventDetails')
 
+    def validate(self):
+        if self.data.get('types'):
```

Comment:
in the context of ebs, it might be better to just put in the valid resource events as an enum in the schema.